EntityFramework.Utilities/Tests/connectionStrings.json

```
@@ -1,4 +1,4 @@
 ﻿{
-	"SqlServer" : "Data Source=./; Initial Catalog=BatchTests; Integrated Security=SSPI; MultipleActiveResultSets=True",
-	"SqlCe" : "Data Source=BatchTests.sdf"
+	"SqlServer": "Server=(localdb)\\MSSQLLocalDB;Integrated Security=true; MultipleActiveResultSets=True",
```

Oops, committed by accident. Also, this isn't used (rather, the connection string is hardcoded in three different contexts.

```suggestion
	"SqlServer" : "Data Source=./; Initial Catalog=BatchTests; Integrated Security=SSPI; MultipleActiveResultSets=True",
```